packages/app/src/app/pages/Sandbox/Editor/Navigation/elements.ts

```diff
@@ -14,66 +15,53 @@ export const Container = styled.div`
     top: ${topOffset}px;
     bottom: ${bottomOffset}px;
     left: 0;
-    display: flex;
-    flex-direction: column;
-    width: 3.5rem;
-    border-right: 1px solid ${theme['activityBar.border'] || 'transparent'};
-    flex: 0 0 3.5rem;
+    padding: ${theme.space[2]}px;
+    border-right: 1px solid ${theme.colors.activityBar.border};
     height: 100%;
-    color: ${theme[`activityBar.inactiveForeground`] ||
-      css`rgba(255, 255, 255, 0.5)`};
+    color: ${theme.colors.mutedForeground};
     font-size: 1.4rem;
-    align-items: center;
-    background-color: ${theme[`activityBar.background`] || css`inherit`};
+    background-color: ${theme.colors.activityBar.background};
   `}
 `;
 
-export const IconContainer = styled.div<{
+export const IconContainer = styled(Stack)<{
   selected: boolean;
   isDisabled: boolean;
-}>`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  transition: 0.3s ease all;
-  height: 3.5rem;
-  width: 3.5rem;
-  font-size: 1.875rem;
-  color: ${props =>
-    props.theme[`activityBar.inactiveForeground`] ||
-    css`rgba(255, 255, 255, 0.5)`};
-  cursor: pointer;
-  background: transparent;
-  border: 0;
-  appearance: none;
-  outline: 0;
+}>(
+  props => css`
+    transition: ${props.theme.speeds[1]}ms ease all;
+    height: ${props.theme.space[9]}px;
+    width: ${props.theme.space[10]}px;
+    color: ${props.theme.colors.activityBar.inactiveForeground};
+    cursor: pointer;
+    background: transparent;
+    border: 0;
+    border-radius: ${props.theme.radii.small}px;
+    box-sizing: border-box;
+    padding: 0;
 
-  &:hover {
-    color: ${props => props.theme[`activityBar.foreground`] || css`white`};
-  }
+    &:hover {
+      background: ${props.theme.colors.sideBar.hoverBackground};
```

Comment:
we should add `activityBar.hoverBackground` to polyfill

Reply:
Done :)